package.json

```diff
@@ -99,16 +99,17 @@
     "node-sass": "^4.5.0",
     "protractor": "^5.1.1",
     "resolve-bin": "^0.4.0",
+    "rollup": "^0.41.6",
     "run-sequence": "^1.2.2",
     "sass": "^0.5.0",
     "selenium-webdriver": "^3.1.0",
     "stylelint": "^7.8.0",
     "travis-after-modes": "0.0.7",
-    "ts-node": "^2.1.0",
+    "ts-node": "^3.0.0",
     "tslint": "^4.4.2",
     "tslint-no-unused-var": "0.0.6",
-    "typescript": "~2.1.6",
-    "uglify-js": "^2.8.7",
+    "typescript": "~2.2.1",
```

Comment:
fyi: just be aware that google3 currently runs on 2.1.x, 2.2.x upgrade is in works.